listener/controller/apibinding_controller_test.go

```diff
@@ -57,23 +58,128 @@ func TestAPIBindingReconciler_Reconcile(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			name:        "workspace_delete_error",
+			clusterName: "dev-cluster",
+			mockSetup: func(ioHandler *workspacefileMocks.MockIOHandler, discoverFactory *discoveryclientMocks.MockFactory, apiSchemaResolver *apischemaMocks.MockResolver, clusterPathResolver *clusterpathMocks.MockResolver) {
```

Comment:
Reduce line length, so it will be easier to read:
```
mockSetup: func(
ioHandler *workspacefileMocks.MockIOHandler, 
discoverFactory *discoveryclientMocks.MockFactory, 
apiSchemaResolver *apischemaMocks.MockResolver, 
clusterPathResolver *clusterpathMocks.MockResolver,
) {
```
Do the same in the similar places.